src/plopp/graphics/canvas.py

```diff
@@ -136,6 +138,14 @@ def autoscale(self, draw: bool = True):
                 self._xmax = max(self._xmax, right.value)
                 self._ymin = min(self._ymin, bottom.value)
                 self._ymax = max(self._ymax, top.value)
+        # TODO: if searching for limits becomes a performance issue, user specified
+        # y-axis limits via vmin/vmax could be taken into account to speed things up.
```

Comment:
Where would you expect a performance issue?

Reply:
My thinking is that in the code above those lines, we do a search on all the lines and collections in the plot for the bounds.
If there are many lines, or if the lines have many points or the collections many patches, this search could get quite slow (this is independent of the changes in this PR).

If the user specifies vmin and vmax for example, this search could just be skipped because we overwrite the values anyway. I'll make the comment clearer.

Reply:
Although that only works for the y-axis limits, and we still need to autoscale for the x-axis...

Reply:
I'll just remove the comment I think.